[32mINFO    [0m root:__init__.py:27 功能的名称为：test_ryvue 功能的描述为：None
[31m[1mERROR   [0m root:test_ryvue.py:70 roleManagement_025断言失败，用例数据为：{'roleName': 'roleManagement_025', 'roleKey': 'roleManagement_025', 'roleSort': 3, 'status': '2', 'menuIds': [], 'deptIds': [], 'menuCheckStrictly': True, 'deptCheckStrictly': True, 'remark': ''}期望数据为：{'msg': '操作失败', 'code': 500}服务器返回的期望数据：{"msg":"操作成功","code":200}
[31m[1mERROR   [0m root:__init__.py:33 断言失败